data/avgi/avgi outfits.txt

```diff
@@ -0,0 +1,608 @@
+	description "Designed to be clustered, this design is capable of full six-axis control."
+
+
+outfit `Z-330 "Spark" Fusion Torch`
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 660000
+	thumbnail "outfit/fusion engine small"
+	"mass" 99
+	"outfit space" -33
+	"engine capacity" -33
+	"thrust" 108
+	"thrusting energy" 9
+	"thrusting heat" 45
+	"thrusting fuel" 0.09
+	"drag reduction" 0.3
+	"flare sprite" "effect/avgi flare/fusion"
+		"frame rate" 15
+		"scale" 0.75
+	"flare sound" "zpinch"
+	"afterburner effect" "small remass injection"
+	"magnozzle rings" 3
+	description "Balance TBD"
+
+
+outfit `Z-660 "Zap" Fusion Torch`
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 1440000
+	thumbnail "outfit/fusion engine medium"
+	"mass" 198
+	"outfit space" -66
+	"engine capacity" -66
+	"thrust" 222
+	"thrusting energy" 18
+	"thrusting heat" 90
+	"thrusting fuel" 0.18
+	"drag reduction" 0.6
+	"flare sprite" "effect/avgi flare/fusion"
+		"frame rate" 15
+	"flare sound" "zpinch"
+	"afterburner effect" "medium remass injection"
+	"magnozzle rings" 6
+	description "Balance TBD"
+
+
+outfit `Z-990 "Arc" Fusion Torch`
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 2760000
+	thumbnail "outfit/fusion engine large"
+	"mass" 297
+	"outfit space" -99
+	"engine capacity" -99
+	"thrust" 360
+	"thrusting energy" 27
+	"thrusting heat" 135
+	"thrusting fuel" 0.27
+	"drag reduction" 0.9
+	"flare sprite" "effect/avgi flare/fusion"
+		"frame rate" 30
+		"scale" 2
+	"flare sound" "zpinch"
+	"afterburner effect" "large remass injection"
+	"magnozzle rings" 9
+	description "The largest of the standard fusion torches used by the Avgi"
+
+
+effect "small remass injection"
+	sprite "effect/avgi flare/afterburner"
+		"frame rate" 30
+		"scale" 0.75
+	sound "flamethrower"
+
+effect "medium remass injection"
+	sprite "effect/avgi flare/afterburner"
+		"frame rate" 30
+	sound "flamethrower"
+
+effect "large remass injection"
+	sprite "effect/avgi flare/afterburner"
+		"frame rate" 30
+		"scale" 2
+	sound "flamethrower"
+
+
+outfit "Remass Injector"
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 2760000
+	thumbnail "outfit/unknown"
+	"mass" 3
+	"outfit space" -3
+	"engine capacity" -3
+	"afterburner thrust" 90
+	"afterburner energy" 3
+	"afterburner heat" -12
+	"afterburner fuel" 0.3
+	"magnozzle rings" -1
+	description "A powerful magnetohydrodynamic turbopump forces a torrent of water into a fusion torch's magnetic nozzle, dramatically increasing thrust at the cost of a significant increase in propellant consumption. The additional mass flow helps reduce the waste heat imparted on a ship by the torch drive."
+	description "The Remass Injector is a component that can be installed into an Avgi fusion torch to give it afterburning capabilities. It requires a magnozzle ring slot to be installed."
+
+
+outfit "Inductive Extractor"
```

Comment:
> So the provided energy offsetting all outfit space requirements assumes that it's producing energy at all times, like an equivalent reactor/ramscoop/etc, but in reality the inductive extractor will only be functioning for small fractions of a ship's flight time. 

That is true. The point about it being enough to offset the fuel, heat and energy costs of the engine still applies though, since they're necessarily active for the same fraction of the flight time.

Reply:
I understand that the whole idea of the Inductive Extractor is to completely offset the need for power generation besides the torches, but they do _very_ odd things in a mixed-engine environment- not a problem right _now_, but possibly with repercussions in the future. Using a Spark with inductive extractors means one can completely offset the power usage of an engine set otherwise composed entirely of Quarg steering and Ionic Afterburners, which I believe would be the most power-hungry engine set in the game. I suspect this would be... rather problematic in player use.